timeless/auth/views.py

```diff
@@ -3,18 +3,36 @@
  methods, once Employee model is available. Fetch the user using Employee model
  and check password hash using Employee.validate_password method. Update html
  templates when methods are implemented. Create more tests for all methods.
-@todo #5:30min Implement before_app_request function that will get the user id
- from session, get user data from db and store it in g.user, which lasts for the
- length of the request. Also, create a decorator that will check, for each view
- if g.user exists and if not, redirect user to login page.
 """
+from functools import wraps
 from flask import (
     Blueprint, flash, g, redirect, render_template, request, session, url_for
 )
+from timeless.employees.models import Employee
 
 bp = Blueprint("auth", __name__, url_prefix="/auth")
 
 
+@bp.before_app_request
+def load_logged_in_user():
+    user_id = session.get("user_id")
+    if user_id is None:
+        g.user = None
+    else:
+        g.user = Employee.query.get(user_id)
```

Comment:
@bkuzmic What will be happened if `user_id` is blank line or for some reason Employee doesn't have this `id`?

Reply:
@vladarefiev It is fine. `Employee.query.get(user_id)` will return `None` for non-existing `user_id`